server/src/main/java/io/druid/server/coordinator/HttpLoadQueuePeon.java

```diff
@@ -261,6 +260,7 @@ public void onSuccess(InputStream result)
             public void onFailure(Throwable t)
             {
               try {
+                responseHandler.description = t.getMessage();
```

Comment:
`t.toString()` is more informative than `t.getMessage()`, since typically it includes the type of the exception but getMessage does not.